src/manifold/deferred.clj

```diff
@@ -1261,11 +1387,11 @@
 
 (extend-protocol Deferrable
 
-  CompletableFuture
+  CompletionStage
   (to-deferred [f]
     (let [d (deferred)]
-      (.handle ^CompletableFuture f
-               (reify java.util.function.BiFunction
+      (.handle ^CompletionStage f
```

Comment:
OK, but rename the param from `f` for Future to something like `cs` for CompletionStage.